### ISSUE TITLE
Ubuntu upstream stream content setup doc

### ISSUE THREAD

**Issue by calancha** [uyuni-project/uyuni-docs#348]
In the 4.0 release used to be a detailed section about how to set up the upstream content for the Ubuntu; that useful section has been dropped.

Also, I see here https://documentation.suse.com/external-tree/en-us/suma/4.1/suse-manager/client-configuration/clients-ubuntu-old.html the following note:
`The mandatory channels do not contain Ubuntu upstream packages. The repositories and channels for synchronizing upstream content must be configured manually.
`
but I cannot find such a note in the docs in out GMC server.

Indeed, with the current documentation installed in our GMC server, I am not able to set up the upstream content; I am forced to look back in older versions of the doc to follow the manual steps referred above.

**Comment by Loquacity** [comment]
Hi @calancha thanks for this, I'm looking into it. Can you please be more specific about which procedure is being excluded? Is it `Procedure: Creating Custom Channels and Repositories`? It looks like this on https://documentation.suse.com/external-tree/en-us/suma/4.0/suse-manager/client-configuration/clients-ubuntu.html

![image](https://user-images.githubusercontent.com/3914967/85968849-7c3bfe80-ba09-11ea-9e65-6b6a634a9e66.png)

**Comment by calancha** [comment]
Exactly, I followed that part in previous Beta testing for 4.1.  It was very clear to follow those steps.
In the GMC documentation that part has disappeared; the reason might be that it is no longer needed (!).
At the current doc, we have the snippet
```
spacewalk-common-channels \
<base_channel_name> \
<client_channel_name> \
<updates_channel_name> 
```
according with my experience testing Ubuntu 16.04, this snippet makes unnecessary the old manual steps; but I am not 100% sure (@juliogonzalez might know this better); if that is the case, and this snippet covers all those manual steps, then one or two sentences mentioning that could be helpful.

I like the following note, found here (https://documentation.suse.com/external-tree/en-us/suma/4.1/suse-manager/client-configuration/clients-ubuntu-old.html):
```
The mandatory channels do not contain Ubuntu upstream packages. The repositories and channels for synchronizing upstream content must be configured manually.
```
I cannot find this note on my GMC test SUMA server.  With this note, and saying that the above snippet takes care of the upstream packages (in case that be true), then I would be happy.

**Comment by juliogonzalez** [comment]
> according to with my experience testing Ubuntu 16.04, this snippet makes unnecessary the old manual steps; but I am not 100% sure (@juliogonzalez might know this better); if that is the case, and this snippet covers all those manual steps, then one or two sentences mentioning that could be helpful.

That is the case, but that command is incorrect, as I told at https://github.com/SUSE/spacewalk/issues/11819

Client channel is needed only for Uyuni, and security channel is missing.

**Comment by calancha** [comment]
Thank you.  I will open tomorrow a PR fixing the wrong parts; I will also add
few sentences to make the text more clear.

**Comment by Loquacity** [comment]
I've addressed SUSE/spacewalk#11819 in PR #349. To avoid confusion, let's close this issue and continue discussion there instead.
